reaper.go

```diff
@@ -140,7 +140,7 @@ func newReaper(ctx context.Context, options ...reaperOption) (*reaper, error) {
 		return nil, fmt.Errorf("listen: %w", err)
 	}
 
-	r.logger.Info("listening", fieldAddress, r.listener.Addr().String())
+	r.logger.Info("Started", fieldAddress, r.listener.Addr().String())
```

Comment:
suggestion: While it's listening here its not processing yet, I would update the following instead and also add a comment as to why the upper case so someone doesn't change at a later date because its inconsistent with everything else which is lower case.:
```go
	r.logger.Info("client processing started")
```

Reply:
But the regular expression that tc-java uses is not camelcase, it expects `.*Started.*`. That's why I added the simple `Started` word, to avoid weird combinations including lowercase and upper case words, like this one:

```go
	r.logger.Info("client processing Started")
```

I'm adding the following comment:
```go
	// This log message, in uppercase, is in use in different Testcontainers libraries,
	// so it is important to keep it as is to not break the current behavior of the libraries.
	r.logger.Info("Started", fieldAddress, r.listener.Addr().String())
```

Reply:
You could swap it round and say:
```go
r.logger.Info("Started client processing")
```

The main point was to have it just before it would actually be able to process requests but it in practice it should be fine either way.